modules/reference/pages/command/command-overview.adoc

```diff
@@ -50,3 +51,9 @@ With these commands, you can manage basic operation of an Open Liberty server.
 == springBootUtility commands
 
 The `springBootUtility` commands help to deploy Spring Boot applications with Open Liberty. You can xref:command/springbootUtility-thin.adoc[create a thin Spring Boot application] and library cache from a fat Spring Boot application. For more information, see link:/guides/spring-boot.html[Containerizing, packaging, and running a Spring Boot application].
+
+[#auditUtility auditReader]
+== auditUtility auditReader command
+
+The `auditUtility auditReader` command is an action of the `auditUtility` command.
+You can run the `auditUtility auditReader` command with parameters to decrypt and unsign logs.
```

Comment:
I would just use the first sentence from the shortdesc, which is more specific:
`The auditUtility auditReader command decrypts and unsigns an audit log that is encrypted, signed, or both.`
then link to the command page for more information (since we dont have any other topics for this command or process)